mautic/mautic
# Add Symphony Messenger Only

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | Y
| Deprecations?                          | N
| BC breaks? (use the c.x branch)        | N
| Automated tests included?              | Y 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes  



#### Description:

I think PR #11180 is too big and it makes it hard for other developers to test and review file changes, I made this new PR to install `symfony/messenger` as a new bundle and install the required packages along with the configuration UI for the new bundle. **Once we get this PR approved and merged, I will make a new PR that will contain messenger alone**



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to the configuration page, check for `Messenger Configuration`
3. Go to `local.php` and see the values of `messenger_dsn` changes according to the settings you fill
4. If you disabled Messenger you should see in `local.php` `messenger_type`  => `sync`
5. Messenger configuration for doctrine variables can be found on https://symfony.com/doc/current/messenger.html#doctrine-transport
